components/mediation/registry/org.wso2.micro.integrator.registry/src/main/java/org/wso2/micro/integrator/registry/MicroIntegratorRegistry.java

```diff
@@ -1005,8 +1005,10 @@ private OMNode readNonXML (URL url) throws IOException {
                 StringBuilder strBuilder = new StringBuilder();
                 try (BufferedReader bReader = new BufferedReader(new InputStreamReader(inputStream))) {
                     String line;
+                    String newLineChar = System.getProperty("line.separator");
```

Comment:
shall we make this as a class level static variable , we might not have to read each time when we execute this method

Reply:
fixed.